README.md

```diff
@@ -1,6 +1,20 @@
-# idena-pocket
-A web wallet for Idena.
+# Idena-pocket
+```Idena-pocket wallet is available at https://pocket.idena.dev```
 
-## Links
-- [Documentation](https://www.idena.dev/idena-pocket)
-- [Telegram](https://t.me/idenadev)
+### Overview 🔎
+Idena-pocket is based on Idena-js library developed by [@idena-dev](https://github.com/idena-dev). 
+Idena-JS is built with typescript.
+
+This wallet does not need an active node needed to run.
+
+As of date, It is using ```rpc.idena.dev``` as the reference endpoint since their is no public rpc endpoint available.
+
+```rpc.idena.dev``` is built and maintained by [@idena-dev](https://github.com/idena-dev/idena-pocket)
+
+### Will this be opensource?
+💯. Feel free to improve on Idena-pocket. Forks and PRs are welcome.
+
+### Useful Links 🌐
+- [Idena-JS documentation](https://www.idena.dev/idena-js/quick-start)
+- [Idena-RPC documentation](https://github.com/idena-dev/idena-rpc)
+- [Idena-pocket website](https://pocket.idena.dev/#/)
```

Comment:
- Idena-JS -> idena-js
- Idena-RPC -> idena-rpc
- Idena-pocket -> idena-pocket

Reply:
Also, I'd like to maintain telegram group link in readme:
- [Telegram](https://t.me/idenadev)